### ISSUE TITLE
Win build does not include pkg-config stuff

### ISSUE THREAD

**Issue by Tobias-Fischer** [conda-forge/libtheora-feedstock#18]
And thus it cannot be found in builds, see e.g. https://dev.azure.com/roboforge/ros_pipelines/_build/results?buildId=221&view=logs&j=741742f9-4d8d-5cb4-d505-fbc77d302dba&t=e2032d63-a24a-53f8-ed77-0549f944863c

/cc @seanyen @wolfv

**Comment by wolfv** [comment]
yeah the issue is that we're building on Win using a CMakeLists ... we could add the necessary bits to generate a pkg-config from it. or patch whoever needs theora to use `find_library`.

**Comment by traversaro** [comment]
We can probably port https://github.com/microsoft/vcpkg/pull/34904 here to solve this.
